tests/src/DictExtraTests.elm

```diff
@@ -0,0 +1,98 @@
+module DictExtraTests exposing (tests)
+
```

Comment:
This looks pretty good. I'd also suggest having a separate test file that copies https://github.com/elm-community/dict-extra/blob/1.3.2/tests/Main.elm and makes whatever the smallest changes are needed to match our context here. Basically, to prove that we pass `Dict.Extra`'s own tests.